packages/devtools_app/lib/src/framework/framework_core.dart

```diff
@@ -19,8 +19,9 @@ import '../shared/primitives/utils.dart';
 import '../shared/scripts/script_manager.dart';
 import '../shared/survey.dart';
 
-typedef ErrorReporter = void Function(String title, dynamic error);
+typedef ErrorReporter = void Function(String title, Object error);
```

Comment:
lints were good at catching a ton of uses of dynamic that were left in our code.